### ISSUE TITLE
[Supporting the platform - Bot Framework Web Chat - Audio]: Caption or a text transcript is not provided for the 'Audio only' content.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#5372]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
Transcript or captions should be provided for the video only or audio only content so that it is easy for the hearing impairment users to understand what is going on in the video or audio and navigate the page without any hindrance.

### Note: 
Login Credentials must not be provided within the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Transcript navigation, Others or unrelated

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type audio word in type your message text box.
4. Navigate to the results.
5. Verify the issue.

### What do you expect?

Caption or a text transcript should be provided for the 'Audio only' content.

### What actually happened?

Caption or a text transcript is not provided for the 'Audio only' content.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/92ee1699-f225-4790-92f5-0cfa2b240e4f
![Image](https://github.com/user-attachments/assets/6476f7f8-a033-4a20-aa09-a4769e23839c)

### MAS reference

MAS 1.2.1 – Audio-only and Video-only (Pre-recorded)

**Comment by OEvgeny** [comment]
@compulim my suggestion would be to deprecate audio/video attachments and provide a guidance of how to allow arbitrary html tags to be included into the activity text.

**Comment by OEvgeny** [comment]
I don't see the browser supports audio captions. There is an options but captions don't show to me in the browser: https://bolt.new/~/sb1-tpv9gzxv. Also have a third-party bug related to #5361

**Comment by Kirank915** [comment]
@OEvgeny Thanks for the update. We will route this bug to third party team via eDAD team as per [External 3rd Party Process](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2f%2fmicrosoft.sharepoint.com%2fteams%2fcea11y%2fSitePages%2fExternal-Bugs-Process---Third-Party-Dependencies.aspx&data=05%7c01%7cv-katkri%40microsoft.com%7c34f1c3338a5c4ca88a0908da4368ce41%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637896414685783444%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=2chKRW7U66oVfw0eCi6G7gu0n1acPjscrCUl6fyEi4A%3D&reserved=0). To route these, we might require some way (code pen or sample environment) where the external team will be able to repro the issue.
We will route these bugs to eDAD team once we have the details.

**Comment by OEvgeny** [comment]
Web Chat team can't fix the issue:
- the above shows the lack of support on the platform level
- the #5361 shows we are missing the options during the Adaptive Card authoring process to address the issue
